plugins/alignments/src/SNPCoverageRenderer/SNPCoverageRenderer.test.js

```diff
@@ -1,63 +0,0 @@
-import SimpleFeature from '@jbrowse/core/util/simpleFeature'
-import SNPCoverageRenderer, { configSchema, ReactComponent } from '.'
```

Comment:
why do we have to delete the SNPCoverageRenderer test?

Reply:
they were in general quite weak (only testing that any imageData object was returned and due to changing API they got removed. I can try to restore these wiggle based tests though